specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/bastionHost.json

```diff
@@ -0,0 +1,440 @@
+             },
+             "privateIPAllocationMethod": {
+               "type": "string",
+               "description": "PrivateIP allocation method.",
+               "enum": [
+                 "Static",
+                 "Dynamic"
+               ],
+               "x-ms-enum": {
+                 "name": "IPAllocationMethod",
+                 "modelAsString": true
+               }
+            }
+          },
+          "description":"Properties of IP configuration of an Bastion Host."
+       },
+       "BastionHostIPConfiguration":{  
+          "properties":{  
+             "properties":{  
+                "x-ms-client-flatten":true,
+                "$ref":"#/definitions/BastionHostIPConfigurationPropertiesFormat",
+               "description":"Represents the ip configuration associated with the resource."
+             },
+             "name":{  
+                "type":"string",
+                "readOnly": true,
+                "description":"Name of the resource that is unique within a resource group. This name can be used to access the resource."
+             },
+             "etag":{  
+                "type":"string",
+                "readOnly": true,
+                "description":"A unique read-only string that changes whenever the resource is updated."
+             },
+             "type": {
+               "readOnly": true,
+               "type": "string",
+               "description": "Ip configuration type."
+             },
+             "id": {
+                "readOnly": true,
+                "type": "string",
+                "description": "Id for The resource"
+             }
+          },
+          "allOf":[  
+             {  
+                "$ref":"./network.json#/definitions/SubResource"
+             }
+          ],
+          "description":"IP configuration of an Bastion Host."
+       },
+       "BastionHostPropertiesFormat":{  
+          "properties":{  
+             "ipConfigurations":{  
+                "type":"array",
+                "items":{  
+                   "$ref":"#/definitions/BastionHostIPConfiguration"
+                },
+                "description":"IP configuration of the Bastion Host resource."
+             },
+             "dnsName":{  
+                "type":"string",
+                "description":"FQDN for the endpoint on which bastion host is accessible."
+             },
+             "provisioningState":{
+               "description":"The provisioning state of the resource.",
+               "$ref":"./network.json#/definitions/ProvisioningState"
+            }
+          },
+          "description":"Properties of the Bastion Host."
+       },
+       "BastionHost":{  
+          "properties":{  
+             "properties":{  
+                "x-ms-client-flatten":true,
+                "$ref":"#/definitions/BastionHostPropertiesFormat",
+                "description":"Represents the bastion host resource."
+             },
+             "name":{  
```

Comment:
I believe `name, type, id` are inherited via
```
          "allOf":[  
             {  
               "$ref": "./network.json#/definitions/Resource"
             }
          ],
```
below. You may wish to keep only properties than you need to override in this case.

Reply:
I had this earlier but i got a CI break. I have reverted to that now. Lets see if CI passes